Cards/Cards.css

```diff
@@ -0,0 +1,212 @@
+    align-items: center;
+    gap: 1rem;
+    flex-wrap: wrap;
+    margin-top: 1rem;
+  }
+  .card-container {
+    width: 40rem;
+    display: flex;
+    flex-direction: column;
+    padding-bottom: 0.5rem;
+    gap: 0.4rem;
+    border-radius: 0.4rem;
+    background-color: var(--card-BG);
+    box-shadow: 4px 4px 15px rgba(0, 0, 0, 0.15);
+  }
+  .card-top-section {
+    display: flex;
+    gap: 1rem;
+    align-items: center;
+    position: relative;
+  }
+  
+  .img-tint {
+    width: 40rem;
+    height: 12.5rem;
+    position: absolute;
+    left: 0;
+    top: 0;
+    background-color: rgba(0, 0, 0, 0);
+  }
+  .img-tint:hover {
+    background-color: rgba(68, 67, 67, 0.08);
+  }
+  
+  .image-box {
+    height: 12.5rem;
+    width: 10rem;
+  }
+  
+  .image-box img {
+    height: 100%;
+    width: 100%;
+    object-fit: fill;
+    border-top-left-radius: 0.4rem;
+    border-top-right-radius: 0.4rem;
+  }
+  
+  .card-details {
+    font-size: 0.8rem;
+  }
+  
+  .card-details h2 {
+    margin-top: 0.4rem;
+    color: var(--secondary-color);
+  }
+  
+  .card-details h4 {
+    margin-top: 0.4rem;
+    font-size: 0.6rem;
+    color: var(--tertiary-color);
+  }
+  .card-bottom-section {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    margin-left: auto;
+    padding-right: 0.5rem;
+  }
+  .primary-solid {
+    background-color: black;
+    border: 1.5px solid black;
+    border-radius: 0.3rem;
+    color: white;
+    padding: 0.8rem 1.5rem;
+  }
+  
+  .primary-outline {
+    background-color: white;
+    border: 1.5px solid black;
+    border-radius: 0.3rem;
+    color: black;
+    padding: 0.8rem 0.8rem;
+  }
+  
+  .card2-container {
+    width: 19rem;
+    text-align: left;
+    background: var(--card-BG);
+    padding-bottom: 0.5rem;
+    border-radius: 0.5rem;
+    box-shadow: 4px 4px 15px rgba(0, 0, 0, 0.15);
+    position: relative;
+    margin: 2rem;
+  }
+  
+  .card2-container,
+  h1,
+  h2,
+  p {
+    margin-left: 1rem;
+    margin-bottom: 0.2rem;
+  }
+  
+  .card2-container h1 {
+    font-size: 1rem;
+    margin-top: 0.7rem;
+  }
+  
+  .card2-container h2 {
+    font-size: 0.8rem;
+    margin-top: 0.4rem;
+    margin-bottom: 1rem;
+    color: var(--secondary-color);
+  }
+  
+  .card2-container h5 {
+    font-size: 0.8rem;
+    margin-top: 0.4rem;
+    margin-bottom: 1rem;
+    padding-left: 1rem;
+    color: var(--secondary-color);
+  }
+  /* card2 details */
+  .card2-container p {
+    font-size: 0.8rem;
+    padding: 0.2rem;
+  }
+  
+  /* card 2 image */
+  .card2-container img {
+    width: 19rem;
+    height: 20rem;
+    border-top-left-radius: 0.5rem;
+    border-top-right-radius: 0.5rem;
+  }
+  
+  .primary-solid-brown {
+    background-color: rgb(66, 27, 27);
+    width: 1.2rem;
+    height: 1.2rem;
+    border-radius: 50%;
+    border: none;
+  }
+  
+  .primary-solid-beige {
+    border-radius: 0.3rem;
+    width: 1.2rem;
+    height: 1.2rem;
+    background-color: rgb(206, 206, 182);
```

Comment:
You can create variable of colors and use them it will help to maintain the code much easier

Reply:
Done. Please review and approve